content/en/Videos/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Videos
+description: Links of gRPC youtube videos
+menu:
+  main: {weight: 4}
+---
+{{< blocks/cover image_anchor="top" height="sm" color="primary" >}}
+{{< page/header >}}
+{{< /blocks/cover >}}
+
+<div class="container l-container--padded">
+
+<div class="row">
+{{< page/toc collapsed=true placement="inline" >}}
+</div>
+
+<div class="row">
+<div class="col-12 col-lg-8">
+
+Following are the links of youtube videos that helped to get familiar with gRPC :
```

Comment:
This is good for a start, but I think it would be nice to have it grouped by type to make it more user-friendly. Something like this:
## Overview and Getting started with gRPC
<The first 4 videos can come here>

## Deep dive into gRPC and its applications
<The last 3 videos can come here>
also some more videos from the channel might be good to add here:
https://youtu.be/7LYvl-nr0t8?feature=shared
https://youtu.be/L6FAtc8N8Vk?feature=shared
https://youtu.be/Z3X6kD_1SFo?feature=shared
https://youtu.be/B8gfu5bs2Pw?feature=shared

## User stories
https://youtu.be/ywrkBqq_LLA?feature=shared
https://youtu.be/gc3kNMvgrHQ?feature=shared
https://youtu.be/GDJrw36wwWY?feature=shared

Note: I've just linked videos, but I meant to add both the videos and descriptions

Reply:
![Screenshot 2024-11-22 2 13 19 PM](https://github.com/user-attachments/assets/655b1d38-576f-4245-b2e9-4be8dbf16494)
![Screenshot 2024-11-22 2 13 37 PM](https://github.com/user-attachments/assets/3a2e401d-b468-4680-b9fd-07b14e7837f1)
![Screenshot 2024-11-22 2 13 51 PM](https://github.com/user-attachments/assets/5a5d659a-16cc-49ad-831f-471c10c15fa1)
![Screenshot 2024-11-22 2 14 00 PM](https://github.com/user-attachments/assets/d3ec1296-a119-4bcf-a068-a38b21e0674b)

Reply:
Changes is done as per comments